nodes/teach_repeat_node.cpp

```diff
@@ -0,0 +1,408 @@
+#include "teach_repeat_node.hpp"
+
+#include <ros/ros.h>
+#include <ros/console.h>
+#include <ros/package.h>
+
+#include "hebi_cpp_api/group_command.hpp"
+#include "hebi_cpp_api/robot_model.hpp"
+
+#include "robot/arm.hpp"
+
+namespace hebi {
+namespace ros {
+
+// Helper function to get index of name from a path or waypoint list
+template <typename NamedItem>
+static int getNameIndex(const hebi_cpp_api_examples::Playback& msg, const std::vector<NamedItem>& named_items, const std::string& name_type) {
+  int index = msg.index;
+  auto name = msg.name;
```

Comment:
I'd do `const auto& name` since I believe this becomes just `std::string` (assuming that's what the type is?) andthe lambdas below copy the string yet again. I may be wrong, but if my assumption is right, this is a very small thing to fix

Reply:
Good catch -- I believe so as well.  I fixed the lambdas, too...